### DISCUSSION TITLE
Welcome to PCjs.org Discussions

### DISCUSSION THREAD

**Discussion by jeffpar** [jeffpar/pcjs#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by disassemble1999** [comment]
Please add simulations for the following:

Windows 98
Microsoft Word 2.0, 4.0 and 6.0
Microsoft Office 95

**Comment by 5c3pT** [comment]
Hi Jeffpar,
I don't know if you read this, I'm new to all this, but I just wanted to let you know I am very pleased to inform you I got PFM working on Windows 10 with DOSbox-X. I am a bit exited and just wanted to send you a quick message. I'm planning om making a D-fend shortcut, but I guess I will have to do some setting of the the appropriate DOSbox version in D-fend, couldn't find the solution this quick, but I will figure it out. THX !!!!
